packages/sdk-metrics/test/export/PeriodicExportingMetricReader.test.ts

```diff
@@ -126,6 +131,52 @@ describe('PeriodicExportingMetricReader', () => {
     sinon.restore();
   });
 
+  const waitForAsyncAttributesStub = sinon.stub().returns(
+    new Promise<void>(resolve =>
+      setTimeout(() => {
+        resolve();
+      }, 10)
+    )
+  );
+  const scopeMetrics: ScopeMetrics[] = [
+    {
+      scope: {
+        name: 'test',
+      },
+      metrics: [
+        {
+          dataPointType: DataPointType.GAUGE,
+          dataPoints: [
+            {
+              // Sample hr time datapoints.
+              startTime: [12345, 678901234],
+              endTime: [12345, 678901234],
+              attributes: {},
+              value: 1,
+            },
+          ],
+          descriptor: {
+            name: '',
+            description: '',
+            unit: '',
+            type: InstrumentType.COUNTER,
```

Comment:
Oh we just dropped this from the type for 2.0, so this won't compile with the latest changes on main
```suggestion
```